### ISSUE TITLE
framework of choice

### ISSUE THREAD

**Issue by madhums** [Cynsar-Capital/cc-app#11]
I am wondering if we can use Ruby On Rails as the framework of choice for this project. It's reliable, has a great community, does a lot of heavy lifting, battle tested with production use at companies like github, shopify, 37signals etc and it's quite mature.

I've been programming with full stack js and I find that the decision on modelling the MC (in MVC) layers is completely left to the people who code. There's a lack of community standard. With Next.js setting new standards, especially with next@13's app router and features like [server actions](https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions-and-mutations) and their [storage solutions](https://vercel.com/storage/postgres), it's becoming harder to maintain a difference between Model, View and clear separation of concerns. 

Is there any willingness to consider Rails as a framework of choice?

**Comment by saranshisatgit** [comment]
Yeah we can but be aware we can always separate the front end but if you think that we can use it as advantage then we should definitely move ahead with the MVC architecture. Though some people use the Rails and Next JS together.

In my opinion we can combine both.
